src/cascadia/TerminalApp/Tab.cpp

```diff
@@ -317,13 +318,13 @@ void Tab::_AttachEventHandlersToControl(const TermControl& control)
 {
     std::weak_ptr<Tab> weakThis{ shared_from_this() };
 
-    control.TitleChanged([weakThis](auto newTitle) {
+    control.TitleChanged([weakThis, this](auto newTitle) {

```

Comment:
Why did you also capture `this` instead of just using the `tab`? (Here and below).

Reply:
I agree, I think we should probably _not_ capture `this` here at all, to make sure we don't accidentally use that instead of the `weakThis.lock()` result